src/plugins/intel_gpu/src/kernel_selector/cl_kernels/mvn_gpu_bfyx_opt.cl

```diff
@@ -57,12 +57,16 @@ KERNEL (mvn_gpu_bfyx_opt)(
     lg_storage[in_data_set_idx] = my_sum;
 
     barrier(CLK_LOCAL_MEM_FENCE);
+    for (uint offset = workers_per_data_set / 2; offset > 0; offset >>= 1) {
```

Comment:
I think it makes sense to make the code more consistent by applying either division by 2 or bitwise shift in a single line